### PR TITLE
LPS-98417 adding package.json to formatSource exception rules

### PR DESCRIPTION
**Pull request by gabrielibson**

### CODE REVIEW

**Comment by gabrielibson** [comment]
Related issue: https://issues.liferay.com/browse/LPS-98417

**Comment by gabrielibson** [comment]
ci:test:sf

**Comment by gabrielibson** [comment]
ci:test:relevant

**Comment by gabrielibson** [comment]
ci:test:relevant

**Comment by natocesarrego** [comment]
Just started reviewing :) <br><br>:octocat: *Sent from [GH](http://nodegh.io).*

File: modules/sdk/project-templates/project-templates-form-field/build.gradle [hunk 1]
```diff
@@ -2,12 +2,22 @@ dependencies {
...
+	compileOnly project(":core:petra:petra-string")
+
+	testCompile group: "biz.aQute.bnd", name: "biz.aQute.bnd", version: "3.5.0"
+	testCompile group: "com.googlecode.java-diff-utils", name: "diffutils", version: "1.3.0"
+	testCompile group: "com.liferay", name: "com.liferay.maven.executor", version: "1.0.0"
+	testCompile group: "junit", name: "junit", version: "4.12"
+	testCompile group: "net.diibadaaba.zipdiff", name: "zipdiff", version: "1.0"
+	testCompile project(path: ":sdk:project-templates:project-templates", configuration: "testAPI")
+
+	testRuntime project(":sdk:project-templates:project-templates").files("classes")
```
Comment (natocesarrego) [inline modules/sdk/project-templates/project-templates-form-field/build.gradle:14]: @gabrielibson, was it necessary due to the update of task `generateSampleProject` or due to the unit tests of the form-field template?

Comment (gabrielibson) [reply modules/sdk/project-templates/project-templates-form-field/build.gradle:14]: @natocesarrego, this dependency: `compileOnly project(":core:petra:petra-string")`, is related to StringPool.BLANK inside of `FormFieldProjectTemplateCustomizer`.

The other ones are related to unit tests of the form-field template. 
I followed [this](https://github.com/natocesarrego/liferay-portal/blob/master/modules/sdk/project-templates/project-templates-service/build.gradle) example suggested by Lawrence.

Thank you for reviewing.

Comment (gabrielibson) [reply modules/sdk/project-templates/project-templates-form-field/build.gradle:14]: @natocesarrego, it's fixed now. Please, let me know if you find any other inconsistency.

Thanks.

**Comment by natocesarrego** [comment]
Hi @gabrielibson,

Thanks for your answer!

As I told you previously, I did some SF to your previous pull, therefore I'd like to send it to you for you add it to this most recent pull, then I'd like to ask you to rerun the command that generated the FAKE GRADLE CACHE commit, to make sure the SF changes will be added to the generated jar of `project-template-form-field`. Lastly, once you did this, can you please send the final pull to https://github.com/gamerson/ and mention me on it?

Thanks for your hard work on it, Gabriel! 🙂

**Comment by natocesarrego** [comment]
Pull request submitted to https://github.com/gabrielibson/liferay-portal/pull/8. [See changes here.](https://github.com/natocesarrego/liferay-portal/compare/2a89e39724f7~2...2a89e39724f7) <br><br>:octocat: *Sent from [GH](http://nodegh.io).*

**Pull request closed without merging**